### PR TITLE
add missing permissions for geo location in UIExplorer Example

### PR DESCRIPTION
**Pull request by offbye**

### CODE REVIEW

File: Examples/UIExplorer/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -3,6 +3,11 @@
           package="com.facebook.react.uiapp" >
 
   <uses-permission android:name="android.permission.INTERNET" />
+  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+  <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
+  <uses-permission android:name="android.permission.CHANGE_WIFI_STATE" />
```
Comment (mkonicek) [inline Examples/UIExplorer/android/app/src/main/AndroidManifest.xml]: Why add these three?

File: Examples/UIExplorer/IntentAndroidExample.android.js [hunk 2]
```diff
@@ -65,6 +65,8 @@ var IntentAndroidExample = React.createClass({
         <OpenURLButton url={'http://www.facebook.com'} />
         <OpenURLButton url={'http://facebook.com'} />
         <OpenURLButton url={'geo:37.484847,-122.148386'} />
+        <OpenURLButton url={'tel:10086'} />
+        <OpenURLButton url={'sms:10086'} />
```
Comment (mkonicek) [inline Examples/UIExplorer/IntentAndroidExample.android.js]: What number is this?

**Comment by Kureev** [comment]
Hi @offbye! To process this PR we need some additional information from you. Would you mind to help us by answering questions above?

**Comment by facebook-github-bot-3** [comment]
@mkonicek would you mind taking a look at this pull request? It's been a while since the last commit was reviewed.

**Comment by mkonicek** [comment]
Closing due to a lack or response from author. See the questions from Nov 20:

> What number is this?
> Why add these three?

**Pull request closed without merging**